apps/nextra/pages/en/build/indexer/indexer-sdk/advanced-tutorials/txn-script.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "Aptos Transaction generated using Move Script"
```

Comment:
I think it might be good to make the title of this consistent with the sidebar title you have in _meta.tsx